src/scss/components/optional/_modal.scss

```diff
@@ -157,6 +154,12 @@ $modal-borderRadius     : 4px;
             }
         }
 
+        .c-modal-content-scrollable {
```

Comment:
Is this missing media queries?

Reply:
Nope, if it is scrollable the class will be applied on the vertical via logic 👍

Reply:
We basically have a bug on menuweb where we want to turn the `max-height` off for one of many modal instances. So the best way is to separate out the scrollability and the heights themselves if that makes sense?